docs/man/docker-cp.1.md

```diff
@@ -11,8 +12,9 @@ CONTAINER:PATH HOSTPATH
 
 # DESCRIPTION
 Copy files/folders from a container's filesystem to the host
-path. Paths are relative to the root of the filesystem. Files
-can be copied from a running or stopped container.
+path running the command. Paths are relative to the root of
```

Comment:
Cool. So, I would recommend being more explicit in the text. These sentences are hard to parse because they look simple but are in fact pretty complicated.  ESL people may find them particularly difficult.  (And isn't the opposite of a relative path an absolute one? So, I might mention that for clarity's sake.) I'd suggest:
----
Copy files or folders from a container's filesystem to the Docker host.  The `PATH` is relative to the container.  The `HOSTPATH` is relative to the host's directory in which `docker cp` is run.  For example, if you are running the command in your home directory, `docker cp foo bar` will create `$HOME/bar`  on your host.  To copy to a specific location, specify an absolute pathname for the HOSTPATH parameter.